src/dql_resolution.erl

```diff
@@ -8,7 +8,9 @@
 %%%-------------------------------------------------------------------
 -module(dql_resolution).
 
--export([resolve/2, propagate/1, start_time/1]).
+% -export([resolve/2, propagate/1, start_time/1]).
+
+-compile([export_all]).
```

Comment:
That's neat, I thought it was only for eunit tests and didn't know it applied to the shell as well